app/controllers/admin/events_controller.rb

```diff
@@ -4,35 +4,40 @@ class EventsController < ApplicationController
     include ApplicationHelper
 
     def index
+      @events = Event.all.order(starts_at: :desc).page(params[:page]).per(20)
```

Comment:
How do we feel about this pagination? Is it too much or too little per page?

Reply:
Luckily it looks like `.all` is ignored here with the pagination but we should probably remove it anyways since in normal circumstances `.all` would pull everything out of the database. Surprised it doesnt in this case, I found this interesting 👇 
```ruby
[55] pry(main)> Event.all.order(starts_at: :desc).page(0).per(1)
  Event Load (8.1ms)  SELECT "events".* FROM "events" ORDER BY "events"."starts_at" DESC LIMIT $1 OFFSET $2  [["LIMIT", 1], ["OFFSET", 0]]
=> []
[56] pry(main)> Event.order(starts_at: :desc).page(0).per(1)
  Event Load (1.7ms)  SELECT "events".* FROM "events" ORDER BY "events"."starts_at" DESC LIMIT $1 OFFSET $2  [["LIMIT", 1], ["OFFSET", 0]]
=> []
```

As for page size, 20 seems good to me.